Add Workshop vignette

CarmenTamayo opened the pull request:
Potential future blogpost with vignette from the 100 days workshop

Bisaloo commented on posts/workshop-vignette/index.qmd:
```diff
@@ -0,0 +1,349 @@
+---
+title: "workshop vignette"
```
Could you make this title "What should the first 100 lines of code written during an epidemic look like?" please?

CarmenTamayo replied:
done

Bisaloo commented on posts/workshop-vignette/index.qmd:
```diff
@@ -0,0 +1,349 @@
+  pdf_document:
+    latex_engine: xelatex
+date: "2023-01-12"
+self_contained: TRUE
```
```suggestion
```

this is handled centrally in the website settings

Bisaloo commented on posts/workshop-vignette/index.qmd:
```diff
@@ -0,0 +1,349 @@
+This vignette summarises the findings from the _100 days and 100 lines of code_ workshop, hosted in December 2022 by [Epiverse-TRACE](https://data.org/news/epiverse-trace-a-values-based-approach-to-open-source-ecosystems/).
+
+
+## What should the first 100 lines of code written during an epidemic look like?
```
```suggestion
```

This is now the title of the post.

Bisaloo commented on posts/workshop-vignette/index.qmd:
```diff
@@ -0,0 +1,349 @@
+output: 
+  pdf_document:
+    latex_engine: xelatex
+date: "2023-01-12"
```
Could you add yourself as author and add some categories (i.e., tags or keywords) please?

Docs: https://quarto.org/docs/websites/website-blog.html#categories

Bisaloo commented on posts/workshop-vignette/index.qmd:
````diff
@@ -0,0 +1,349 @@
+
+<br>
+```{r, echo=FALSE, out.width="100%"}
+knitr::include_graphics('posts/workshop-vignette/Scenario1smq.png')
````
```suggestion
knitr::include_graphics('Scenario1smq.png')
```

Paths in Rmarkdown (`.Rmd`) and Quarto (`.qmd`) documents are relative to the document where they are called. So here, relative to `index.qmd`. Since your images are in the same folder as `index.qmd`, you should call them directly.

CarmenTamayo commented:
Hi Hugo, I have changed the folder name and added all the changes you suggested, apart from the tags because I wasn't sure if this is related to the 'categories' section on the link you shared- are those what I should add? Thank you

Bisaloo reviewed (commented):
Thanks, looks good! I've made some suggestions for the post metadata (title, author, date).

Could you also please add an alt-text to all images for visitors who use a screen reader?

- [Why we need to add alt-text?](https://www.howtogeek.com/821673/what-is-alt-text-why-you-should-use-it-and-how-to-write-it/)
- [How to add an alt-text in quarto?](https://quarto.org/docs/authoring/figures.html#alt-text)

Alternatively, the figures could be converted to actual tables, from which text can be read & copied (we can figure out how to color each cell to keep the current look).

Bisaloo commented:
Ideally, if you feel the need to retranscribe the entire text, you should probably turn the images into actual text, or a table as mentioned in the previous message.

My main issue here is that users who for some reason can't see the images will miss the entire presentation of the scenarios. And all the rest of the text doesn't make sense if you don't know the scenarios.

If it's too much work to convert the images to text and you can't do it, then yes, you should probably just include a simple caption.

CarmenTamayo commented:
Okay, I will add the contents of the images as alt text if that is okay, I’m getting it done today 😊

From: Hugo Gruson ***@***.***>
Date: Wednesday, 8 March 2023 at 09:02
To: epiverse-trace/epiverse-trace.github.io ***@***.***>
Cc: Carmen Tamayo Cuartero ***@***.***>, Assign ***@***.***>
Subject: Re: [epiverse-trace/epiverse-trace.github.io] Workshop vignette (PR #9)

Ideally, if you feel the need to retranscribe the entire text, you should probably turn the images into actual text, or a table as mentioned in the previous message.

My main issue here is that users who for some reason can't see the images will miss the entire presentation of the scenarios. And all the rest of the text doesn't make sense if you don't know the scenarios.

If it's too much work to convert the images to text and you can't do it, then yes, you should probably just include a simple caption.

—
Reply to this email directly, view it on GitHub<https://github.com/epiverse-trace/epiverse-trace.github.io/pull/9#issuecomment-1459793038>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/A3GJNZAPIIS3Z2P6HZP6GZ3W3BDKPANCNFSM6AAAAAAUUAFSU4>.
You are receiving this because you were assigned.Message ID: ***@***.***>
